arithmetization/src/main/java/net/consensys/linea/zktracer/module/blake2fmodexpdata/BlakeModexpDataOperation.java

```diff
@@ -167,7 +167,7 @@ private void traceModexpResult(Trace trace, final UnsignedByte stamp) {
 
   private void commonTrace(Trace trace, UnsignedByte stamp, int index, Bytes input, int indexMax) {
     trace
-        .stamp(stamp)
+        .stamp(stamp.toInteger())
```

Comment:
Not sure this is the right way to do this.

Reply:
looking at `UnsignedByte.java` it seems ok to me. It's used to convert to BigInt to